jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -144,56 +199,134 @@ def _recompute_mark_positions(self, viewer):
         viewer_mark = self._get_mark(viewer)
         if not np.any(in_viewer):
             viewer_mark.x, viewer_mark.y = [], []
-            return

```

Comment:
what's the benefit to getting rid of this and replacing it with elses?  It seems that nothing else needs to happy in this case anyways besides setting to the empty list